### DISCUSSION TITLE
Here's a BG image for your map.

### DISCUSSION THREAD

**Discussion by josh-allison** [vonbeardy/hd2galaxy-issues#17] [Show and tell]
![HD2GMBG](https://github.com/vonbeardy/hd2galaxy-issues/assets/78192304/0c9af5bc-f51b-494e-9f0b-615e54f24eb9)

This is what it looks like in action on your site ❤️ (as of now).
![image](https://github.com/vonbeardy/hd2galaxy-issues/assets/78192304/7c74750e-c0c0-4353-8a45-aee1bf66ef06)

**Comment by vonbeardy** [comment]
I created an issue (https://github.com/vonbeardy/hd2galaxy-issues/issues/19) to track this. I'll hopefully get this added tonight!
